test/test_performance.py.in

```diff
@@ -37,15 +37,21 @@ def _raw_to_csv(csv_data, csv_path):
     appears on the plot.
     """
     dataframe = pd.read_csv(csv_data, skiprows=21, sep='[ \t]*,[ \t]*', engine='python')
-    dataframe_agg = dataframe.agg(['max', 'mean', 'sum'])
+    dataframe_agg = dataframe.agg(['max', 'mean', 'sum', 'median'])
 
+    percentils_latency = dataframe['latency_mean (ms)'].describe(percentiles=[0.95])
```

Comment:
Great to see P95 here. What is the title of the P95 measure on the graph?

Reply:
Added p95 to latency test plot